### ISSUE TITLE
Few suggestions (i can try to do the PR if needed)

### ISSUE THREAD

**Issue by littleski** [samchon/prisma-markdown#20]
* (Description) Why not writing the type for each field, as well a props (FK, nullable, Unique, etc...) ? They are in the ERD, but the ERD doesn't embed the comments. Maybe having a config field top level or a specific markup per model to choose that ?
* Is there a way, when using multiple workspaces to only have a synthetic "version" of a table in a workspace. For example, i want to clearly see the relations with the "user" table, but i don't really care of the objects related in the "User" workspace, i just want to see the relations, each other workspace, the same way, will fully document their domain, but in theses erd, the table "user" will show a synthetic version ? (Ideally, Markdown links to the right domain position would be amazing)
* Is there a way to "Group" the workspace in the table of content ? or sort them alphabetically for instance, so it's easyer to find love when you have 20+ workspaces ?

❤️

**Comment by samchon** [comment]
1. It seems good suggestion
2. I'm not sure what you're talking about, is your idea not expressible with the existing `@namespace`, `@erd` and `@describe`?
3. It would better to support more configurable options in the prisma schema file (`generation markdown { ... }`)

**Comment by samchon** [comment]
Welcome your PR.

**Comment by littleski** [comment]
Yeah. Haven't got many time lately, and encountered a few TS issues is the
existing code. Have priorised putting all the doc for one project to have a
good test base and be able to iterate. Found some issue in our schéma by
doing so, business first. :)


Le jeu. 8 févr. 2024 à 11:53, Jeongho Nam ***@***.***> a
écrit :

**Comment by samchon** [comment]
Close due to too much long time no response
